package.json

```diff
@@ -153,7 +152,6 @@
     "url-loader": "^4.1.1",
     "vite": "^6.2.4",
     "vitest": "^3.0.3",
-    "vue-eslint-parser": "^10.1.1",
```

Comment:
We need this as we explicitly use it in the config. It does not work without flat config (the v9 version of the config).

Reply:
🙁

Reply:
But at least `sanitize-filename` is fine.